lib/subscription/subscription.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path='../../typings/tsd.d.ts' />
+
+import BAPI = require('../blpapi-wrapper');
+
+export = Subscription;
+
+class Subscription extends BAPI.Subscription {
+
+    correlationId: number;
+
+    constructor( cid: number,
```

Comment:
Remove the space: `constructor( cid: number,` => `constructor(cid: number,` and indent remaining arguments accordingly.